src/components/Summary/AlterationSummary.vue

```diff
@@ -47,8 +47,8 @@
           </v-col>
 
           <v-col cols="8" class="mt-n1">
-            <div class="company-name font-weight-bold">{{ companyName }}</div>
-            <div class="company-name">{{ getNameRequest.nrNumber }}</div>
+            <div class="company-name font-weight-bold text-uppercase">{{ companyName }}</div>
```

Comment:
FYI there are 2 parts to this feature:
1. convert the input to uppercase
2. CSS to show caps even when they enter lower case; yes, point 1 covers this but there's a bit of delay and the CSS covers it